### PR TITLE
general README updates, as well intermediate DoD cert directions

### PR DESCRIPTION
**Pull request by dandds**
This updates the README to refer to the scripts-to-rule-them-all and adds directions for adding DoD intermediate certs to the CA bundle. This will change pending resolution of [this](https://www.pivotaltracker.com/n/projects/2160940/stories/158497124) Pivotal discussion.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,37 +4,44 @@
 
 ## Back end local development
 
-* Start the stack with Docker Compose:
-
-```bash
-docker-compose up -d
-```
+To build, run `script/sertup`.  `script/server` starts the stack with Docker Compose:
```
Comment (briandds) [inline README.md]: `sertup` -> `setup`?

Comment (ddsdevon) [reply README.md]: Er mah gerd, sertup!

Comment (dandds) [reply README.md]: whoops, fixed

**Comment by dandds** [comment]
Rebased to get the test passing after merging https://github.com/dod-ccpo/authnid/pull/19. I knew something was up when a README change broke CI...

File: README.md [hunk 2]
```diff
@@ -4,37 +4,44 @@
...
+The tests run with Pytest. Modify and add tests to `./tests/`.
+
+### SSL Certificates for Testing
 
-- the server's certificate authority (`ssl/certificate-authority/ca.crt`), set in the testing docker container with teh `REQUESTS_CA_BUNDLE` env variable
-- the client public cert and private key, both in `ssl/client-certs/`
+The `ssl` directory contains example certs for configuring the regular server SSL, client authentication, and the client certificates currently written to the sample PIVKey. It also contains a script, `make-certs.sh`, which can be used to write a new certificate authority and sign a CSR for the server SSL. The script writes the CSR to list multiple valid hosts (`subjectAltName`) so that the final cert can work across environments (like docker, for instance, where it's host name is "backend").
 
-The tests run with Pytest, modify and add tests to `./tests/`.
+note: add info about the intermediate CAs
```
Comment (jordangov) [inline README.md]: looks like we have the note?

**Review by jordangov — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (dandds) [reply README.md]: yup, fixed.

**Review by jordangov — APPROVED** [review]

**Pull request merged**